sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/HiveMetastoreLazyInitializationSuite.scala

```diff
@@ -61,11 +63,10 @@ class HiveMetastoreLazyInitializationSuite extends SparkFunSuite {
         spark.sql("show tables")
       })
       for (msg <- Seq(
-        "show tables",
         "Could not connect to meta store",
         "org.apache.thrift.transport.TTransportException",
         "Connection refused")) {
-        exceptionString.contains(msg)
+        assert(exceptionString.contains(msg))
```

Comment:
amazing...

Reply:
How about use `checkError` with `_LEGACY_ERROR_TEMP_3065`?

Reply:
Hi @beliefer, these assertions are not for _LEGACY_ERROR_TEMP_3065 but the root cause.